src/Sylius/Bundle/CoreBundle/Fixture/Factory/AdminUserExampleFactory.php

```diff
@@ -124,7 +129,8 @@ private function createAvatar(AdminUserInterface $adminUser, array $options): vo
         $imagePath = $this->fileLocator->locate($options['avatar']);
         $uploadedImage = new UploadedFile($imagePath, basename($imagePath));
 
-        $avatarImage = new AvatarImage();
+        /** @var AvatarImage $avatarImage */
+        $avatarImage = $this->avatarImageFactory->createNew();
```

Comment:
Once `$this->avatarImageFactory` will be nullable, we need to fallback to `new AvatarImage()` if the factory will not be inserted